app/models/library_staff_record.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+# This class is responsible for storing and retrieving relevant
+# metadata from the library_staff_record table in the database
+class LibraryStaffRecord < ApplicationRecord
+  scope :query, lambda { |search_term|
+                  where('last_name ILIKE :search OR first_name ILIKE :search OR middle_name ILIKE :search ' \
```

Comment:
This query seems like it would only match on single-word queries.  For example, if a LibraryStaff has a first name of "Tiberius", middle name of "The", and Last name of "Cat", I think a search for "Tiberius the Cat" wouldn't match.  Maybe this is future work, though.

Reply:
Yeah, I should have put a note. It was a initial search until I had a better idea of what we would need for searching the records.